templates/definition/tariff/octopus-productcode.yaml

```diff
@@ -20,7 +20,15 @@ params:
     help:
       de: "Die DNO-Region, in der Sie sich befinden. Weitere Informationen: https://www.energy-stats.uk/dno-region-codes-explained/"
       en: "The DNO region you are located in. More information: https://www.energy-stats.uk/dno-region-codes-explained/"
+  - name: directDebit
+    type: bool
+    required: false
+    default: True
+    help:
+      de: "Auf 'False' setzen, wenn Sie Ihre Rechnung NICHT per Lastschrift bezahlen."
```

Comment:
I'd prefer a less verbose description:

"Rechnung wird per Lastschrift bezahlt"

Reply:
If you've got better German to contribute then go from it, it's not my first language by any means! 😵‍💫

Reply:
Can we do the same non-verbose style in english?

    Bill collected with Direct Debit

Reply:
Yeah, the EN can likely be compacted. Let me think about that one...